### PR TITLE
Fix warning message for paged attributes and remote storage

### PR DESCRIPTION
**Pull request by hmusum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by arnej27959 — DISMISSED** [review]
we should probably cap the list if it's too long, to avoid unreadable message

File: config-model/src/main/java/com/yahoo/vespa/model/application/validation/PagedAttributesRemoteStorageValidator.java [hunk 1]
```diff
@@ -56,11 +57,12 @@ private static void validatePagedAttributes(DeployLogger logger, String clusterN
                 .formatted(clusterName, join(fields)));
     }
 
-    private static String join(Set<Attribute> fields) {
-        return fields.stream()
+    private static String join(List<Attribute> fields) {
+        var ret = fields.stream()
```
Comment (arnej27959) [inline config-model/src/main/java/com/yahoo/vespa/model/application/validation/PagedAttributesRemoteStorageValidator.java:61]: add `.limit(10)` to actually limit also?

Comment (hmusum) [reply config-model/src/main/java/com/yahoo/vespa/model/application/validation/PagedAttributesRemoteStorageValidator.java:61]: Thanks, f**ked up commit, so had to redo it, apparently lost the most important line when doing that.

**Review by arnej27959 — APPROVED** [review]
👍

**Pull request merged**